CHANGELOG.md

```diff
@@ -1,4 +1,35 @@
-# [0.0.0](https://github.com/hamlet-io/engine-plugin-aws/compare/v8.0.0...v0.0.0) (2021-01-11)
+# [0.0.0](https://github.com/hamlet-io/engine-plugin-aws/compare/v8.0.1...v0.0.0) (2021-03-30)
```

Comment:
Should this be looking at engine-plugin-azure not engine-plugin-aws?

Reply:
Fixed, here and on all the other repos I copied the same file to 🤦